assignments/closure.js

```diff
@@ -1,5 +1,16 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
+function outside(){
+  var outty = 10;
```

Comment:
```suggestion
  let outty = 10;
```
var isn't block scoped